### ISSUE TITLE
HTTPerror with Python2

### ISSUE THREAD

**Issue by SirJohnDoe** [twinone/gtst-downloader#1]
I'm getting this traceback when running the script:

```
Traceback (most recent call last):
  File "download.py", line 95, in <module>
    content = urllib2.urlopen(url).read()
  File "/usr/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 407, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 520, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 445, in error
    return self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 379, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 528, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
urllib2.HTTPError: HTTP Error 400: Bad Request
```

I've already checked out the playlist files and everything, and the method appears to be working just fine. I tried Python3 as well, but it seems that various functions need to be updated first (e.g. print).

**Comment by twinone** [comment]
What's the url it's requesting?

**Comment by SirJohnDoe** [comment]
I used this token: 88ccb525-4e3a-3153-97ee-5cef127ea666

Which leads to www.rtlxl.nl/#!/expeditie-robinson-187039/88ccb525-4e3a-3153-97ee-5cef127ea666

Just figured out that it's because the manifest URL in the download.py refers to GTST, so I changed it and now it works. Thanks for this tool! Would be a pita to do it manually.

EDIT: it seems to work now, except I don't know where the resulting MP4 is supposed to show up. I already changed the 'SERVE_FILE' to another directory, because my system doesn't have /var/www. The gtst_downloader_out folder in /tmp is empty when the script says 'done'.
